coinstacks/ethereum/api/src/index.ts

```diff
@@ -24,7 +26,15 @@ app.get('/health', async (_, res) => {
   })
 })
 
-app.use('/docs', swaggerUi.serve, async (_req: Request, res: Response) => {
+const options = {
+  customCss: '.swagger-ui .topbar { display: none }',
+  customSiteTitle: 'ShapeShift API Docs',
```

Comment:
Wondering if we want to put coin name into the title as well. Thoughts?

Reply:
Will include coinstacks in title